test/runtests.jl

```diff
@@ -493,7 +492,8 @@ end
     #Index with range stride much larger than chunk size
     a = _DiskArray(reshape(1:100, 20, 5, 1); chunksize=(1, 5, 1))
     @test a[1:9:20, :, 1] == trueparent(a)[1:9:20, :, 1]
-    @test getindex_count(a) == 3
+    # now getindex_count(a) == 1
```

Comment:
Doesn't this mean sparse ranges will now read all the data in some cases?

I thought the ideas of `batchgetindex` was to exlicitly just read the required chunks.